### ISSUE TITLE
Populate Source Fields

### ISSUE THREAD

**Issue by samm82** [JacquesCarette/Drasil#997]
As mentioned in #424, the Source fields need to be populated with citations in the examples.

## TODO
- [x] HGHC (Probably doesn't need them, but I'll double check)
- [x] GlassBR
- [x] NoPCM
  - [x] TMs
  - [x] GDs
- [x] SWHS
  - [x] TMS
  - [x] GDs
- [x] SSP
  - [x] GD7
- [x] GamePhysics

**Comment by samm82** [comment]
Also, the Source, Description and Ref. By fields are missing in GD7 of SSP - should I assume that the Source is the same as GD8, and what should be done about the other two fields? A new issue?

![image](https://user-images.githubusercontent.com/35857611/44054283-2ec7c21c-9f10-11e8-8415-15488d235bbf.png)

**Comment by samm82** [comment]
Also @smiths, in caseStudies, the Source fields for GamePhysics are completely empty. Should this be a separate issue as well?

**Comment by JacquesCarette** [comment]
I think it is safe to assume the Source and Ref. By for GD7 are the same as for GD8.

**Comment by samm82** [comment]
Then should I reference GD7 in the description of GD8, and add a description ("See GD8") to GD7?

**Comment by JacquesCarette** [comment]
We do allow labels to be URLs. I don't quite remember the details of how to do that -- you might need to read the Haskell source. It should at least be possible to create it. Then try `makeRef` and see what happens.

**Comment by JacquesCarette** [comment]
No, because GD7 doesn't depend on GD8. You should 'reference' the same Source.  And does DD17 and IM4 reference GD7?

It is weird that GD7 have no description.

**Comment by samm82** [comment]
DD17 doesn't reference either of them:
![dd17](https://user-images.githubusercontent.com/35857611/44102541-f182df10-9fb7-11e8-9b75-e125c44b33b8.png)

And IM4 doesn't explicitly reference either of them, but it uses both of their equations:

![im4](https://user-images.githubusercontent.com/35857611/44102621-27b43f66-9fb8-11e8-9828-4d8456ca5d16.png)

---

And what I meant about referencing GD7 - the description of GD8 says "These equations...", but only one equation is defined, so I'm assuming the same description applies to GD7 (x-component) and GD8 (y-component), so this connection should be captured in the description of GD7.

In Drasil, the description of both of these GDs have the standard list of quantities used, but the Notes of GD8 contain the Description from caseStudies, but I think it should still reference GD7.

**Comment by JacquesCarette** [comment]
I probably have things backwards as far as referencing.

Also, I get this feeling that GD7 and GD8 should be a single definition, not two. [This may not currently be possible.]

**Comment by samm82** [comment]
It currently isn't possible to have two equations defined in a DD - what should the fix be for now?

**Comment by JacquesCarette** [comment]
Try to make both be as similar as possible.

**Comment by samm82** [comment]
@JacquesCarette What should the shortname be for a URI link? Should it be the full web address or should it be abbreviated (ie. "http://www.efunda.com/formulae/heat_transfer/conduction/overview_cond.cfm" vs. "efunda.com")

**Comment by samm82** [comment]
@JacquesCarette And an update - Clicking on one of these links created with `makeRef` just appends the URI to the end of the current file path (eg. pathToSRS.html#www.full.com/web/address)

**Comment by JacquesCarette** [comment]
If at all possible, the shortname for a URI link should be 'meaningful' (because it is supposed to be seen by humans) rather than URL-like.  Something closer to the title of the page being linked to, rather than something like the URL.

**Comment by samm82** [comment]
I'm going to create a separate issue for GamePhysics, and then this issue can be closed. Who should I assign the issue to? @JacquesCarette

**Comment by JacquesCarette** [comment]
Right now, no one. @smiths and I will need to review all of those in early September and figure out what to do with them.

**Comment by samm82** [comment]
#1005
